### PR TITLE
Run shortcircuit analysis

### PR DESCRIPTION
**Pull request by etiennehomer**

### CODE REVIEW

File: src/components/shortcircuit-analysis-result.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
```
Comment (FranckLecuyer) [inline src/components/shortcircuit-analysis-result.js:2]: 2022

**Review by FranckLecuyer — CHANGES_REQUESTED** [review]

**Review by FranckLecuyer — CHANGES_REQUESTED** [review]

File: src/translations/fr.json [hunk 2]
```diff
@@ -152,6 +153,11 @@
     "sensitivityAnalysisResults": "Résultats de l'analyse de sensibilité",
     "sensitivityAnalysisError": "Une erreur est survenue lors de l'exécution de l'analyse de sensibilité",
 
+    "ShortCircuitAnalysis": "Analyse de court-circuit",
+    "ShortCircuitAnalysisParameters": "Paramètres d'analyse de court-circuit",
+    "ShortCircuitAnalysisResults": "Résultats d'analyse de court-circuit",
```
[the remark this answers is not in the document]
Comment (etiennehomer) [reply src/translations/fr.json]: Done

File: src/components/shortcircuit-analysis-result.js (hunk 1 above)
Comment (etiennehomer) [reply src/components/shortcircuit-analysis-result.js]: done

File: src/redux/actions.js [hunk 3]
```diff
@@ -268,6 +268,18 @@ export function resetSensiNotif() {
...
 
+export const ADD_SHORT_CIRCUIT_NOTIF = 'ADD_SHORT_CIRCUIT_NOTIF';
+
+export function addShortCircuitNotif() {
+    return { type: ADD_SHORT_CIRCUIT_NOTIF };
+}
+
+export const RESET_SHORT_CIRCUIT_NOTIF = 'RESET_SHORT_CIRCUIT_NOTIF';
+
+export function resetShortCircuitiNotif() {
```
[the remark this answers is not in the document]
Comment (etiennehomer) [reply src/redux/actions.js]: done

File: src/utils/rest-api.js [hunk 4]
```diff
@@ -884,6 +884,80 @@ export function fetchSensitivityAnalysisResult(studyUuid, currentNodeUuid) {
...
+        (response) =>
+            response.ok
+                ? response
+                : response.text().then((text) => Promise.reject(text))
+    );
+}
+
+export function stopShortCircuitAnalysis(studyUuid, currentNodeUuid) {
+    console.info(
+        'Stopping sensitivity analysis on ' +
```
[the remark this answers is not in the document]
Comment (etiennehomer) [reply src/utils/rest-api.js]: done

File: src/utils/rest-api.js [hunk 5]
```diff
@@ -884,6 +884,80 @@ export function fetchSensitivityAnalysisResult(studyUuid, currentNodeUuid) {
...
+            return response.text();
+        } else {
+            return Promise.resolve(0);
+        }
+    });
+}
+
+export function fetchShortCircuitAnalysisResult(studyUuid, currentNodeUuid) {
+    console.info(
+        'Fetching short circuit analysis on ' +
```
[the remark this answers is not in the document]
Comment (etiennehomer) [reply src/utils/rest-api.js]: done

File: src/components/app-top-bar.js [hunk 6]
```diff
@@ -33,18 +33,21 @@ import {
...
     addLoadflowNotif,
     addSANotif,
     addSensiNotif,
+    addShortCircuitNotif,
     centerOnSubstation,
     openNetworkAreaDiagram,
     resetLoadflowNotif,
     resetSANotif,
     resetSensiNotif,
+    resetShortCircuitiNotif,
```
[the remark this answers is not in the document]
Comment (etiennehomer) [reply src/components/app-top-bar.js:50]: done

File: src/utils/rest-api.js [hunk 7]
```diff
@@ -884,6 +884,80 @@ export function fetchSensitivityAnalysisResult(studyUuid, currentNodeUuid) {
     });
 }
 
+export function startShortCircuitAnalysis(studyUuid, currentNodeUuid) {
+    console.info(
+        'Running short circuit analysis on ' +
```
Comment (laurgarn) [inline src/utils/rest-api.js:889]: Could use backticks string: `Running short circuit analysis on ${studyUuid] and node ${currentNodeUuid} ...`

Comment (etiennehomer) [reply src/utils/rest-api.js:889]: done

**Review by FranckLecuyer — APPROVED** [review]

**Pull request merged**